### ISSUE TITLE
"No Tiller RC" message could be cleaned up

### ISSUE THREAD

**Issue by mboersma** [skippbox/cabin-issues#6]
Maybe reword the "Cancel | OK" dialog when Tiller isn't found to something like this:

> **No Tiller Found**
> Cabin needs Tiller in order to deploy services.
> Do you want Cabin to create a Tiller replication controller in this cluster?

(Also let me know if this isn't the kind of feedback you're looking for. I haven't found any actual bugs yet. 😄)

**Comment by runseb** [comment]
yes perfect feedback.

the helm work is known not to work yet. but hopefully we will get it to work soon.

**Comment by sebgoa** [comment]
This was fixed
